.github/workflows/ci-x.yml

```diff
@@ -51,3 +51,5 @@ jobs:
         run: mvn clean install -B -Pquarkus,auth-server-quarkus -DskipTests -f testsuite/pom.xml
       - name: Run base tests
         run: mvn clean install -B -Pquarkus,auth-server-quarkus -f testsuite/integration-arquillian/tests/base/pom.xml | misc/log/trimmer.sh; exit ${PIPESTATUS[0]}
+      - name: Run cluster tests
+        run: mvn clean install -B -Pauth-server-cluster-quarkus -Dsession.cache.owners=2 -Dtest=**.cluster.** -f testsuite/integration-arquillian/pom.xml  | misc/log/trimmer.sh; exit
```

Comment:
This should be a separate job

Reply:
Why?

Reply:
Maybe because you want to run them in parallel. I've changed the workflow as per your comment.